syncapi/storage/postgres/syncserver.go

```diff
@@ -776,19 +990,25 @@ func (d *SyncServerDatasource) getStateDeltas(
 			if membership := getMembershipFromEvent(&ev.Event, userID); membership != "" {
 				if membership == gomatrixserverlib.Join {
 					// send full room state down instead of a delta
-					var s []streamEvent
+					var s []types.StreamEvent
 					s, err = d.currentStateStreamEventsForRoom(ctx, txn, roomID, stateFilterPart)
 					if err != nil {
 						return nil, nil, err
 					}
+					/*
```

Comment:
What are we going to do with this code? I don't like the idea of commented out code that stays indefinitely in the code base.